pygit2/settings.py

```diff
@@ -101,4 +101,30 @@ def cache_object_limit(self, object_type, value):
         """
         return option(GIT_OPT_SET_CACHE_OBJECT_LIMIT, object_type, value)
 
+    def __set_ssl_cert_file(self, value):
+        """Set the ssl cert file. The value cannot be read.
+        """
+        return option(GIT_OPT_SET_SSL_CERT_LOCATIONS, value, None)
```

Comment:
Setters don't return values.

Reply:
When setting one value we shouldn't reset another. If a user wants to do this they can update it to `None` explicitly.